### PR TITLE
adapt upgrade nodes command to the api

### PR DESCRIPTION
**Pull request by MaximilianMeister**
in the beginning it was planned to upgrade nodes one by one, but
now the api just allows the upgrade of all nodes at once

### CODE REVIEW

**Comment by MaximilianMeister** [comment]
[#hackathon - adapt crowbarctl upgrade nodes call from single to all nodes](https://trello.com/c/LZaxPsTE/152-hackathon-adapt-crowbarctl-upgrade-nodes-call-from-single-to-all-nodes)

**Comment by jsuchome** [comment]
To be precise: the method will still do the upgrade of nodes one by one (but user can't choose the order)

File: lib/crowbar/client/app/upgrade.rb [hunk 1]
```diff
@@ -184,18 +184,16 @@ def crowbar
...
         end
 
-        desc "node NODE",
-          "Upgrade a single node"
+        desc "nodes",
+          "Trigger the upgrade on all nodes"
 
         long_desc <<-LONGDESC
-          `node NODE` will upgrade the operating system of a single node.
+          `nodes` will upgrade the operating system on all nodes.
```
Comment (jsuchome) [inline lib/crowbar/client/app/upgrade.rb]: Not just operating system, also the SOC product.

File: lib/crowbar/client/command/upgrade/nodes.rb [hunk 2]
```diff
@@ -32,7 +32,7 @@ def execute
             request.process do |request|
               case request.code
               when 200
-                say "Successfully triggered node upgrade on #{args.id}"
+                say "Successfully triggered node upgrade on all nodes"
```
Comment (jsuchome) [inline lib/crowbar/client/command/upgrade/nodes.rb]: I'd say "... triggered the upgrade of the nodes"

**Pull request merged**